grails-app/domain/org/pih/warehouse/inventory/OutboundStockMovement.groovy

```diff
@@ -339,4 +340,8 @@ class OutboundStockMovement implements Serializable, Validateable {
         return requisition?.approvalRequired && origin?.approvalRequired && RequisitionStatus.compare(requisition.status, RequisitionStatus.PICKING) == -1
     }
 
+    // This has to be named with the get prefix to align with the StockMovement DTO
```

Comment:
you can't name it `isReturn()` and just call the full method (with parentheses) in the gsp?